test/e2e/service.go

```diff
@@ -266,10 +270,6 @@ var _ = Describe("Services", func() {
 				CreateExternalLoadBalancer: true,
 			},
 		}
-
-		By("cleaning up previous service " + serviceName + " from namespace " + ns)
```

Comment:
Very cool. Thanks Derek. I'll take your new code for a spin next week.
On Mar 28, 2015 4:37 AM, "Derek Carr" <notifications@github.com> wrote:

> In test/e2e/service.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6125#discussion_r27344404>
> :
>
> > @@ -266,10 +270,6 @@ var _ = Describe("Services", func() {
> >  				CreateExternalLoadBalancer: true,
> >  			},
> >  		}
> > -
> > -		By("cleaning up previous service " + serviceName + " from namespace " + ns)
>
> FYI - Namespace deletion was merged
> … <#14c602c550f7356e_>
> Sent from my iPhone
>  On Mar 27, 2015, at 9:03 PM, Quinton Hoole <notifications@github.com>
> wrote: In test/e2e/service.go: > @@ -266,10 +270,6 @@ var _ =
> Describe("Services", func() { > CreateExternalLoadBalancer: true, > }, > }
> > - > - By("cleaning up previous service " + serviceName + " from namespace
> " + ns) Done in commit 24f4561
> <https://github.com/GoogleCloudPlatform/kubernetes/commit/24f456159c31085f3a73cea7e8eb38945a3511be>
> — Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6125/files#r27344404>
> .
>